src/scss/components/_scroll-indicator.scss

```diff
@@ -1,37 +1,60 @@
 .scroll {
   background-color: $white;
-  display: block;
-  margin-bottom:76px;
+  margin-bottom: 76px;
   padding: 20px 0 0 0;
 
   &__button {
+    align-items: center;
     align-self: center;
     border: none;
     display: flex;
-    height: 40px;
-    margin:auto;
+    flex-direction: column;
+    margin: auto;
     width: 40px;
   }
 
-  &__button:focus {
-    outline: none;
-  }
-
   &__text {
     display: none;
     font-family: $hkgrotesk-regular;
     font-size: $mobile-card-titles;
   }
 
   @include viewport-desktop() {
+    &__button {
+      height: 85px;
+      width: 339px;
+    }
+
     &__text {
       display: block;
+      margin-bottom: 100px;
       text-align: center;
     }
+
+    &__image {
+      background-color: transparent;
+      width: 40px;
+    }
+
+    &__bounce {
```

Comment:
This could be a modifier of the `__button`, as it is part of the component state or behavior